k8s/redis-ha/README.md

```diff
@@ -145,16 +145,17 @@ the selected version.
 > over time.
 
 ```shell
-export TAG="5.0"
+export TAG="6.2"
+export REDIS_EXPORTER_TAG="1.3.1"
 ```
 
 Configure the container image:
 
 ```shell
 export IMAGE_REGISTRY="marketplace.gcr.io/google"
 
-export IMAGE_REDIS="${IMAGE_REGISTRY}/redis-ha"
-export IMAGE_REDIS_EXPORTER="${IMAGE_REGISTRY}/redis-ha/redis-exporter:${TAG}"
+export IMAGE_REDIS="${IMAGE_REGISTRY}/redis-ha:${TAG}"
+export IMAGE_REDIS_EXPORTER="${IMAGE_REGISTRY}/redis-ha/redis-exporter:${REDIS_EXPORTER_TAG}"
```

Comment:
Revert to k8s app repo `"${IMAGE_REGISTRY}/redis-ha/redis-exporter:${TAG}"` as discussed